app/views/groups/index.html.haml

```diff
@@ -1,10 +1,7 @@
 - content_for(:title) do
   = t :groups_title
 
-= inject_enterprises
-
-:javascript
-  angular.module('Darkswarm').value('groups', #{render partial: "json/groups", object: @groups})
+= inject_groups
```

Comment:
:+1: ideally this would be a new api endpoint, not in scope for this PR.

Reply:
Yes, I wanted to shift some of these ams injection parts to the API, but there were some problems when I tried it with /shops for example. We can do it at some point.

Reply:
Spec added: https://github.com/openfoodfoundation/openfoodnetwork/pull/3833/commits/d9bafd56daaca7e98d030ae5205d15be7e8bcf55#diff-81148c208b0c1bdbfcd8e77f3fc82e57